priv/static/Linux.sh

```diff
@@ -23,6 +23,9 @@ cyan='\033[0;36m'
 
 function already_installed() {
     case $1 in
+    "Git")
```

Comment:
Merge main branch into this one. I guess you cherry-picked or rebased, which you shouldn't do in this case.

Reply:
I rebased. What should I do whenever I want to update my branch but there are conflicts?

Reply:
Merge, and fix any conflicts that occur.

Reply:
Okay.